app/test_main.py

```diff
@@ -1,3 +1,28 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected_human_age",
+    [
+        (0, 0, [0, 0]),
+        (16, 16, [1, 1]),
+        (25, 28, [2, 2]),
+        (14, 14, [0, 0]),
+        (44, 48, [7, 6]),
```

Comment:
You should cover all test cases

if function receives data out of normal range, such as negative numbers  or really large numbers;
if the function receives an incorrect type of data it raises the correct exception.